configure.ac

```diff
@@ -1842,6 +1842,45 @@ if test "x$enable_packet_ring" != "xno" ; then
 	AC_SUBST(PCAP_SUPPORT_PACKET_RING)
 fi
 
+AC_ARG_ENABLE([rdma],
+[AC_HELP_STRING([--enable-rdma],[enable RDMA capture support @<:@default=yes, if support available@:>@])],
+    [],
+    [enable_rdmasniff=ifavailable])
+
+if test "xxx_only" = yes; then
+	# User requested something-else-only pcap, so they don't
+	# want D-Bus support.
```

Comment:
yes, you can guess where I copy and pasted that code snipped from :)

Fixed in the updated commit.